README.md

```diff
@@ -11,12 +11,17 @@ A debian-based linux distribution
 ## Role Variables
 
 
-| Name              | Default/Required | Description                                 |
-|-------------------|:----------------:|---------------------------------------------|
-| `tftpd_username`  | `tftp`           | User under which the tftp daemon runs       |
-| `tftpd_directory` | `/srv/tftp`      | Path where the served files reside          |
-| `tftpd_address`   | `:69`            | Adress on which the daemon should listen on |
-| `tftpd_options`   | `--secure`       | Additional options to pass to the daemon    |
+| Name              | Default/Required | Description                                   |
+|-------------------|:----------------:|-----------------------------------------------|
+| `tftpd_username`  |      `tftp`      | User under which the tftp daemon runs         |
```

Comment:
Please don't center the columns using spaces. This is already done using markdown rendering, but it bloats commits and is changed every time you or someone else on the team changes the same readme.